docscan/sandbox/request/check/id_document_comparison_check.go

```diff
@@ -0,0 +1,55 @@
+package check
+
+import (
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check/report"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"
+)
+
+// IDDocumentComparisonCheck Represents a document authenticity check
+type IDDocumentComparisonCheck struct {
+	*documentCheck
```

Comment:
We should embed `check` here instead as comparison checks don't have a document filter

```suggestion
	*check
```